test/views/propertyDetails/html/propertyDetailsProfessionallyValuedSpec.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.propertyDetails.html
+
+import forms.PropertyDetailsForms
+import play.api.data.{Form, FormError}
+import play.twirl.api.Html
+import utils.viewHelpers.AtedViewSpec
+
+class propertyDetailsProfessionallyValuedSpec extends AtedViewSpec {
```

Comment:
can you move the test to html package please

Reply:
Moved it